### ISSUE TITLE
error "Cannot insert (magit-lfs :key ":" :description "Magit-LFS") into magit-dispatch"

### ISSUE THREAD

**Issue by gonewest818** [Ailrun/magit-lfs#3]
After installing magit-lfs I've noticed this message in the logs:
```
Cannot insert (magit-lfs :key ":" :description "Magit-LFS") into magit-dispatch; (0 3 -1) not found
```

I found I can fix this by changing the setup of the ":" key like this:
```
(transient-append-suffix 'magit-dispatch '(0 2 -1)
  `(magit-lfs
    :key ,magit-lfs-suffix
    :description "Magit-LFS"))
```

with my reasoning being, the first section of the magit transient has three columns, therefore `(0 2 -1)` would put the ":" entry at the end of the third column.  I believe `(0 -1 -1)` might work also.

**Comment by Ailrun** [comment]
Thank you for the posting. I will fix this soon with `(0 -1 -1)`.

**Comment by Ailrun** [comment]
Fixed in 566684fd49bbf908e7be686d7811711360ae36ac
